### PR TITLE
Release v0.6.0

### PR DESCRIPTION
**Pull request by roidelapluie**
Signed-off-by: Julien Pivotto <roidelapluie@inuits.eu>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
...
+## 0.5.1 / 2021-01-15
+
+This release includes a bugfix for a side-channel security issue that would
+allow an attacker to verify if a user is defined in the configuration by timing
+request. #39
+
+* [ENHANCEMENT] Cache basic authentication results to significantly improve
+  performance. #32
+* [BUGFIX] Prevent user enumeration by timing requests. #39
+
```
Comment (SuperQ) [inline CHANGELOG.md:14]: This looks like it's duplicating the previous release.

**Comment by SuperQ** [comment]
Want to `make update-go-deps` as well?

**Comment by roidelapluie** [comment]
done

**Review by SuperQ — APPROVED** [review]
LGTM

**Pull request merged**